lib/vcr/test_frameworks/rspec.rb

```diff
@@ -5,27 +5,28 @@ module RSpec
     module Metadata
       extend self
 
+      def vcr_cassette_name_for(metadata)
+        description = if metadata[:description].empty?
```

Comment:
Minor: Perhaps we can use a newline after the assignment `=`, to reduce the amount of indentation used?

```ruby
description =
  if meta....
    # ...
  else 
    # ...
  end
```

Reply:
Sure thing.

Reply:
Thanks!